# Labelled alternative (CHANGELOG #4e, batch3_class_decisions #1):
#   bd_doxa_programas_HGPE_v2.xlsx -> bd_doxa_programas.xlsx ; bd_doxa_candidatos_HGPE.xlsx -> bd_doxa_candidatos.xlsx
--- deposit_xlsx_names
+++ run_v2_xlsx_names
@@ -1,4 +1,3 @@
 HGPE_lei.xlsx
-bd_doxa_candidatos_HGPE.xlsx
-bd_doxa_programas_HGPE.xlsx
-bd_doxa_programas_HGPE_v2.xlsx
+bd_doxa_candidatos.xlsx
+bd_doxa_programas.xlsx
